tests/test_api.py

```diff
@@ -174,7 +174,6 @@ def test_no_issues(self):
         result = json.loads(self.app.post('/query', data=json.dumps({
             'project': 1,
             'granularity': 3600,
-            'issues': [],
```

Comment:
All of the column references to to `issue` in this file will just be rewritten to be a reference to `group_id`:

https://github.com/getsentry/snuba/blob/e2fdee78046d4673c19a4d2cbd964919621f0ec7/snuba/datasets/events.py#L212-L213